keyboards/ymdk/ymd09/keymaps/andys8/keymap.c

```diff
@@ -0,0 +1,81 @@
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+
+enum custom_keycodes {
+    GIT_STASH = SAFE_RANGE,
+    GIT_STASH_POP,
+    GIT_COMMIT,
+    COPY_PASTE,
+    E_INFO,
+    E_WARN,
+    E_TABLE,
+    E_SHRUG
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case E_INFO:
+            if (record->event.pressed) {
+                send_unicode_string("ℹ️");
+            }
+            break;
+        case E_WARN:
+            if (record->event.pressed) {
+                send_unicode_string("⚠️");
+            }
+            break;
+        case E_TABLE:
+            if (record->event.pressed) {
+                send_unicode_string("(╯°□°)╯︵ ┻━┻");
+            }
+            break;
+        case E_SHRUG:
+            if (record->event.pressed) {
+                send_unicode_string("¯\\_(ツ)_/¯");
+            }
+            break;
+        case GIT_STASH:
+            if (record->event.pressed) {
+                send_string("git stash ");
+                tap_code(DE_MINS);
```

Comment:
If you include `sendstring_german.h`, you should be able to use a single function.

Reply:
Thanks for this great hint. Makes everything a lot better.